Dockerfile

```
@@ -0,0 +1,37 @@
+FROM python:3.9-slim as base
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y \
+    curl \

```

Really minor but if we're going to line break here I'd instinctively break on the commands rather than mid-command:
```suggestion
RUN apt-get update \
    && apt-get install -y curl \
```